fc/mef.py

```diff
@@ -539,54 +261,67 @@ def sc_fun(p,x):
         return np.exp(p[0] * x + p[1])
     
     # Fit parameters
-    err_par = lambda p: err_fun(p, peaks_ch, peaks_mef)
+    err_par = lambda p: err_fun(p, fl_channel, fl_mef)
     res = minimize(err_par, params, options = {'gtol': 1e-6})
 
     # Separate parameters
-    sc_params = res.x
+    beads_params = res.x
 
     # Beads model function
-    sc_beads = lambda x: fit_fun(sc_params, x)
+    beads_model = lambda x: fit_fun(beads_params, x)
 
     # Standard curve function
-    sc = lambda x: sc_fun(sc_params, x)
+    std_crv = lambda x: sc_fun(beads_params, x)
     
-    return (sc, sc_beads, sc_params)
-
-def plot_standard_curve(peaks_ch, 
-                        peaks_mef,
-                        sc_beads,
-                        sc_abs,
-                        xlim = (0., 1023.),
-                        ylim = (1, 1e8)):
+    return (std_crv, beads_model, beads_params)
+
+def plot_standard_curve(fl_channel,
+                        fl_mef,
+                        beads_model,
+                        std_crv,
+                        xlim=(0.,1023.),
+                        ylim=(1.,1e8)):
```

Comment:
Not a fan of these hardcoded defaults for `xlim` and `ylim`. Don't like referencing 1023 which is specific to our cytometer bit depth or `1e8` which I have no idea where it comes from. I think these values should ~~either not have defaults (forcing the user to specify them), or~~ default to whatever `matpotlib` decides to show (extremes of `fl_channel` and `fl_mef`? which seems appropriate to me if you have no other additional knowledge about what the limits should be), e.g. `=None` and ignore them.